sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.students.controller.getAssessmentNOA.e2e-spec.ts

```diff
@@ -41,7 +42,7 @@ describe("AssessmentStudentsController(e2e)-getAssessmentNOA", () => {
     db = createE2EDataSources(dataSource);
   });
 
-  it("Should get the student NOA details for an eligible application when the student tries to access it.", async () => {
+  it("Should get the student NOA details for an eligible part-time application when the student tries to access it.", async () => {
```

Comment:
The ticket is scope is full-time applications and hence I would recommend to use a full-time test to validate them and not touch any part-time applications.

Reply:
I can still see the part-time test validating the new fields. 

<img width="550" height="537" alt="image" src="https://github.com/user-attachments/assets/8916e16c-ae0b-47ec-8118-295d08ffba8c" />